src/foam/u2/detail/SectionedDetailPropertyView.js

```diff
@@ -201,7 +201,7 @@ foam.CLASS({
         .addClass(`sectioned-detail-property-${this.prop.name}`)
         .add(this.slot(function(mode, prop, prop$label) {
 
-          var errorSlot = prop.validateObj && prop.validationTextVisible ?
+          var errorSlot = prop.required != false && prop.validateObj && prop.validationTextVisible ?
```

Comment:
shouldn't required property property be handled in the validateObj function?

Reply:
we overwrite validateObj for each property, so we would need to update all of them. this is a more generic solution.